src/workloads/query/SlidingWindows.yml

```diff
@@ -377,6 +377,674 @@ Actors:
                   default: "Not available"
                }}}}}]
 
+- DerivativeTimeBased:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: DerivativeTimeBased
+      active: [23]
+      operationMetricsName: DerivativeTimeBased
+      nopInPhasesUpTo: *MaxPhases
+      Pipeline:
+          [{$setWindowFields: {
+            partitionBy: "$partitionKey",
+            sortBy: {time: 1},
+            output: {rate: {
+              $derivative: {input: "$temp", unit: "second"},
```

Comment:
I wonder why some of the units here are seconds and some are hours. We just need to make sure the range is not too small to be not meaningful.

Reply:
I think i made a mistake here. I wanted to have around 100-200 documents which comes to around 20 min interval. I mistakenly put 20 sec here.

Reply:
Two existing tests had 1 hour range. which amounts to around 500 docs. This should be ok. I will change all to 1 hour range.

Reply:
Sounds good.